### PR TITLE
Added conditionals for related card, move CSS to cards.scss

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: assets/scss/cards.scss [hunk 1]
```diff
@@ -1,3 +1,94 @@
...
+  a {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .card-img {
+    background-position: center center;
+    background-size: cover;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
```
Comment (monkemedia) [inline assets/scss/cards.scss]: Could we use shorthand here? `border-radius: 4px 4px 0 0`

File: components/generic/ContentCard.vue [hunk 2]
```diff
@@ -1,8 +1,10 @@
 <template>
   <b-card
-    class="mb-4 text-left"
+    class="mb-4 text-left content-card"
     data-qa="content card"
     no-body
+    :class="isRelated && 'related-card'"
```
Comment (monkemedia) [inline components/generic/ContentCard.vue]: Can you explain whats going on here? 'related-card' will only show if 'isRelated' equals true?

Comment (mirjamverloop) [reply components/generic/ContentCard.vue]: Yes, indeed. See also https://michaelnthiessen.com/conditional-class-binding-vue#guard-expressions

File: assets/scss/cards.scss (hunk 1 above)
Comment (mirjamverloop) [reply assets/scss/cards.scss]: Good idea, I have updated it to a shorthand version.

File: components/generic/ContentCard.vue (hunk 2 above)
Comment (monkemedia) [reply components/generic/ContentCard.vue]: I wonder if it's better to take the standard approach which I think is easier to read

`:class="{ 'related-card' : isRelated }"`

**Pull request merged**